.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+### What was the expected behaviour?
+
+Please detail what the expected behaviour was
+
+### What happened instead?
+
+Please detail what was the actual behaviour
+
+### Version info:
+
+- Onfido SDK
+- OS
+- Browser
+- Device
+- Node
+- Framework or relevant dependencies (if applicable)
+
+### Integration configuration:
+
+Please add code snippet(s) of your integration with the Onfido SDK and all customisations of it.
+
+### Steps to reproduce:
+
+Please describe in as much detail as possible on how to reproduce the problem
+
+### Demo:
+If possible, please share a url or snippet with a working example for us to reproduce the issue
```

Comment:
Minor: 
- I think URL instead of "url" is better since it is an abbreviation
- for consistency should use "code snippet" instead of "snippet"

Reply:
also you have 
Please detail what the expected behaviour was
vs
Please detail what was the actual behaviour

i would say change Please detail what **was** the actual behaviour to Please detail what the actual behaviour **was**